webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -91,7 +91,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
   });
   const [sortString, setSortString] = useState<string>('');
   const [experiments, setExperiments] = useState<Loadable<ExperimentWithTrial>[]>(
-    Array(page * PAGE_SIZE).fill(NotLoaded),
+    Array(PAGE_SIZE).fill(NotLoaded),
```

Comment:
with paginated view, this would fill some large numbers potentially (e.g. page = 4 will have 400 not-loaded rows). Also for page 0 of infinite scroll this ends up being 0 at the start